lib/gooddata/lcm/actions/synchronize_user_filters.rb

```diff
@@ -42,6 +42,67 @@ class SynchronizeUserFilters < BaseAction
         description 'Restricts synchronization to specified segments'
         param :segments_filter, array_of(instance_of(Type::StringType)), required: false
 
+        description 'Organization Name'
+        param :organization, instance_of(Type::StringType), required: false
+
+        description 'Domain'
+        param :domain, instance_of(Type::StringType), required: false
+
+        description 'DataProduct to manage'
+        param :data_product, instance_of(Type::GdProductType), required: true
+
+        description 'Segments to manage'
+        param :segments, array_of(instance_of(Type::SegmentType)), required: false
+
+        description 'DataLogger'
+        param :gdc_logger, instance_of(Type::GdLogger), required: true
+
+        description 'GDC Project'
+        param :gdc_project, instance_of(Type::GdProjectType), required: false
+
+        description 'GDC Project Id'
+        param :gdc_project_id, instance_of(Type::StringType), required: false
+
+        description 'User brick users'
+        param :users_brick_users, instance_of(Type::ObjectType), required: false
+
+        description 'Fail Early'
+        param :fail_early, instance_of(Type::BooleanType), required: false
+
+        description 'Strict'
+        param :strict, instance_of(Type::BooleanType), required: false
+
+        description 'Username'
+        param :username, instance_of(Type::StringType), required: false
+
+        description 'Password'
+        param :password, instance_of(Type::StringType), required: false
+
+        description 'AWS Client'
+        param :aws_client, instance_of(Type::GdSmartHashType), required: false
+
+        description 'Input source of the Users Brick. Needed to prevent ' \
+                    'deletion of filters for a user that is to be removed.'
+        param :users_brick_config, instance_of(Type::UsersBrickConfig), required: true
+
+        description 'Development Client Used for Connecting to GD'
+        param :development_client, instance_of(Type::GdClientType), required: true
+
+        description 'GDC client protocol'
+        param :client_gdc_protocol, instance_of(Type::StringType), required: false
+
+        description 'GDC client hostname'
+        param :client_gdc_hostname, instance_of(Type::StringType), required: false
+
+        description 'GDC password'
+        param :gdc_password, instance_of(Type::StringType), required: false
+
+        description 'GDC username'
```

Comment:
these params aren't referenced in this action, why do they need to be specified?